frontend/scss/components/molecules/level-toggle.scss

```diff
@@ -138,7 +138,31 @@ $borderSize: 5px;
     }
   }
 
+  .info-button {
+    input[type=checkbox]{
+      position: absolute;
+      top: 0;
+      right: 0;
+      width: 50px;
+      height: 50px;
+      opacity: 0;
+      z-index: 10;
+      &:focus ~ .#{molecule('tip')} {
+          display: block;
+      }
+    }
+    &:focus {
+      outline: none;
+    }
+    &:hover {
+      .#{molecule('tip')} {
+        display: block;
+      }
+    }
+  }
+
```

Comment:
The info button is really nice and works well with the mouse over.
Can we extract the info-button, so that we have a reusable component?
This would also reduce the css size somewhat.